foundations/javascript_basics/revisiting_rock_paper_scissors.md

```diff
@@ -30,22 +30,18 @@ When you don’t need a branch anymore it can be deleted using `git branch -d <b
 
-For example: if you have a bug in a new feature you’re working on that you can’t figure out, and it causes your code to break, you don’t want to commit that broken code and have it in your project’s “permanent record”. Thankfully, branches make it easy to share code on GitHub without having to commit problematic code where it can be seen in the future!
-
-1. Make a new branch (called `temp`) and change to it with `git checkout -b temp`
-2. Commit the current state of your code to your `temp` branch like you normally would
-3. Push your `temp` branch to your GitHub repo with `git push origin temp`
-4. You (or anyone else with the link to your repo!) can now select your `temp` branch using the branch selector dropdown.
-
-<img width="602" alt="Dropdown menu of branches on GitHub" src="https://cdn.statically.io/gh/TheOdinProject/curriculum/32a651388da10d018d4143b066badf2c9f27dc93/foundations/javascript_basics/revisiting_rock_paper_scissors/imgs/00.png">
+For example: if you have a bug in a new feature you’re working on that you can’t figure out, and it causes your code to break, you don’t want to commit that broken code and have it in your project’s “permanent record”. You could instead create a new temporary branch, switch to it and commit your code to this new branch. If you then push this new temporary branch to GitHub you can share it with others that may be able to help solve your problem. In the assignment below you will get some hands-on practice on making new branches. 
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
 1. Set up a new branch on your previous Rock Paper Scissors repo
    1. Since we'll be making a UI for our Rock Paper Scissors game, make a new branch and change to it with the command `git checkout -b rps-ui`.  
-   2. You are now working in the `rps-ui` branch, locally. However, this branch does not exist in your remote repo yet. If you go to your github repo page, you'll see that you only have 1 branch, which would be `main`. Let's push this new branch to your remote repo with the command `git push origin rps-ui`. Now, you'll see two branches in your Github repository!  
+   2. You are now working in the `rps-ui` branch, locally. However, this branch does not exist in your remote repo yet. If you go to your github repo page, you'll see that you only have 1 branch, which would be `main`. Let's push this new branch to your remote repo with the command `git push origin rps-ui`. Now, you'll see two branches in your Github repository! You can select the new branch on GitHub using the dropdown branch selector shown in the screenshot below.
+   
+   <img width="602" alt="Dropdown menu of branches on GitHub" src="https://cdn.statically.io/gh/TheOdinProject/curriculum/32a651388da10d018d4143b066badf2c9f27dc93/foundations/javascript_basics/revisiting_rock_paper_scissors/imgs/00.png">
```

Comment:
**thought (non-blocking):** The old screenshot shows a `master` branch instead of `main` branch. We could update the image to keep consistent with the lesson and our curriculum's usage of `main`. 

Additionally we could format the new image as recommended in our new Layout Style Guide:https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#images

This is non-blocking on the pull request though! Just let me know if you'd like to add that change or not. And thanks again for all the help with this lesson 🙏

Reply:
@dm-murphy when I get a chance I’ll look through all of it and implement what’s needed and probably suggested as well. 

Thank you as well for all the work you do for TOP.